src/main/scala/com/gu/subscriptions/cas/Configuration.scala

```diff
@@ -0,0 +1,17 @@
+package com.gu.subscriptions.cas
+
+import com.typesafe.config.{Config, ConfigFactory}
+
+object Configuration {
+  val appConfig: Config = ConfigFactory.load()
+
+  val proxy = appConfig.getString("proxy")
+
+  val (proxyScheme, proxyHost, proxyPort):(String,String,Int) = {
+    val urlParser = "(\\w+)://([^:/]+):?(\\d+)?/?".r
```

Comment:
Is this really the best way for us to parse a url? Can we not use `spray.http.uri` ? https://github.com/spray/spray/blob/release/1.2/spray-http/src/main/scala/spray/http/Uri.scala